wordle-web/src/scripts/wordsService.ts

```diff
@@ -7,9 +10,63 @@ export abstract class WordsService {
     return this.#words.includes(word)
   }
 
-  static validWords(): Array<string> {
-    //Todo
-    return new Array<string>()
+  static validWords(
```

Comment:
Display the number of valid words based on the entered letters ❌✅
I interpreted this as an autocomplete, so if someone was started to type "BA" it would show them all the valid words that start "BA". I'm not sure which one they actually want though.

Reply:
I believe since the method is to get all the valid words based on guesses, that your currently entered "non-guessed" letters probably aren't included in the result, since you shouldn't know their state to filter against until they've been guessed... although technically I guess it does say "entered -letters-"